src/config.rs

```diff
@@ -92,16 +92,24 @@ pub struct ConfigGeneral {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct WindowSettings {
 	pub sidebar_width: f64,
+	pub window_size: (f64, f64),
+}
+
+impl Default for WindowSettings {
+	fn default() -> Self {
+		WindowSettings {
+			sidebar_width: DEFAULT_SIDEBAR_WIDTH,
```

Comment:
I'd do the same here as well and use the `WindowSettings::default` for `DEFAULT_SIDEBAR_WIDTH` no?

Reply:
This is used for double click snap back to default.

Reply:
Which you could just use the default as well no?

Reply:
I guess so. Just feels odd to create a window settings::default().sidebar_width

Reply:
It's more semantic? Says what it does?